src/components/GlobalSearch/GlobalSearch.js

```diff
@@ -37,7 +36,11 @@ export default class GlobalSearch extends React.Component {
           id="doc-search"
           placeholder="Search"
         />
+        <Close20
```

Comment:
I think this may need to be wrapped in a button... I'm having issues trying to close the search when clicking on the icon. Making it a button will also make it keyboard accessible, so a keyboard user can tab to the close and close it that way

Reply:
Using a `blur` event on the search component would also make sure that the search closes whenever a keyboard user leaves the search in either direction. MDN has a similar component that functions this way. 

Might be an alternative to managing the click event listener on the document as well.